documentation/docs/06-service-workers.md

```diff
@@ -7,6 +7,13 @@ Service workers act as proxy servers that handle network requests inside your ap
 In SvelteKit, if you have a `src/service-worker.js` file (or `src/service-worker.ts`, or `src/service-worker/index.js`, etc) it will be built with Vite and automatically registered.
 
 > You can change the location of your service worker in your [project configuration](#configuration-files).
+<br />
+You can also provide the `scope` for the service worker, by default `/`. You **MUST** modify you `manifest.json` file
+to include also the `scope`: if you are using the default `scope` value, then you need to configure `scope: '/'`.
```

Comment:
please don't manually wrap the lines. just let it be one long line and then the text editor and browser can do word wrap

Reply:
And how we can solve this?